src/run_program.rs

```diff
@@ -1172,15 +1170,15 @@ const TEST_CASES: &[RunProgramTest] = &[
     RunProgramTest {
         prg: "(softfork (q . 161) (q . 0) (q . (q . 42)) (q . ()))",
         args: "()",
-        flags: ENABLE_BLS_OPS,
+        flags: 0,
         result: None,
         cost: 10000,
         err: "softfork specified cost mismatch",
     },
 
     // without the flag to enable the BLS extensions, it's an unknown extension
     RunProgramTest {
-        prg: "(softfork (q . 161) (q . 0) (q . (q . 42)) (q . ()))",
+        prg: "(softfork (q . 161) (q . 1) (q . (q . 42)) (q . ()))",
```

Comment:
extension 0 is not unknown anymore. It's known as the BLS/coinid operators extension.
This test is specifically testing an *unknown* extension, so we use 1 now instead, which is still unknown